hub/util/tests/test_slice.py

```diff
@@ -0,0 +1,46 @@
+from hub.util.slice import merge_slices
+
+
+RANGE = range(100)
+
+
```

Comment:
Can we add a test for verifying the slice content as well?

Reply:
I'm unsure what you mean, can you clarify? If you're asking for a case that verifies the content of the resulting `slice` object, note that comparing the slices is equivalent to comparing the indexed result on a range. There is an isomorphic relationship between slice contents and the resulting indexes.

Reply:
I agree and tests should verify that. I would suggest having at least some tests to verify the relation and everything is working as expected.

Reply:
I'm not sure you understand what I am saying. This is not a relation that is enforced, it is a fundamental property of the slices. These are mathematically equivalent, there's no reason to test anything else. I'm not going to test against a language feature working properly.

Reply:
Can we add more edge test cases? Or are you against that as well?